checker/line_length_checker_test.go

```diff
@@ -41,10 +42,12 @@ func TestLineLengthChecker(t *testing.T) {
 				TabWidth:  4,
```

Comment:
I don't see why we need a table test here. Am I missing something?

Reply:
I removed it.